tests/test_examples.py

```diff
@@ -262,48 +261,60 @@ def run_qh_step(n, eq):
 
 @pytest.mark.regression
 @pytest.mark.solve
-@pytest.mark.xfail
 def test_qh_optimization1():
     """Tests precise QH optimization, step 1."""
     eq0 = load(".//tests//inputs//precise_QH_step0.h5")[-1]
     eq1 = load(".//tests//inputs//precise_QH_step1.h5")
     eq1a = run_qh_step(0, eq0)
-    rho_err, theta_err = area_difference_desc(eq1, eq1a)
-    np.testing.assert_allclose(rho_err, 0, atol=1e-4)
-    np.testing.assert_allclose(theta_err, 0, atol=1e-4)
+    rho_err, theta_err = area_difference_desc(eq1, eq1a, Nr=1, Nt=1)
+    # only need crude tolerances here to make sure the boundaries are
+    # similar, the main test is ensuring its not pathological and has good qs
+    assert rho_err.mean() < 1
+
+    grid = LinearGrid(M=eq1a.M_grid, N=eq1a.N_grid, NFP=eq1a.NFP, sym=False, rho=1.0)
+    data = eq1a.compute(["|B|_mn", "B modes"], grid, M_booz=eq1a.M, N_booz=eq1a.N)
+    idx = np.where(np.abs(data["B modes"][:, 1] / data["B modes"][:, 2]) != 1)[0]
+    B_asym = np.sort(np.abs(data["|B|_mn"][idx]))[:-1]
+    np.testing.assert_array_less(B_asym, 1e-1)
```

Comment:
This isn't actually a good test for QS because of Issue #370. But we can fix this test as part of PR #399.